packages/devtools_app/lib/src/shared/table/_table_row.dart

```diff
@@ -219,13 +219,71 @@ class _TableRowState<T> extends State<TableRow<T>>
   bool isActiveSearchMatch = false;
 
   bool isHovering = false;
+  late FixedExtentDelegate rowExtentDelegate;
+  late List<_TableRowPartDisplayType> rowDisplayParts;
+  List<_TableRowPartDisplayType> rowDisplayPartsHelper() {
+    final rowDisplayParts = <_TableRowPartDisplayType>[];
+    final groups = widget.columnGroups;
+    if (groups != null && groups.isNotEmpty) {
+      for (int i = 0; i < groups.length; i++) {
+        final groupParts = List.generate(
+          groups[i].range.size as int,
+          (index) => _TableRowPartDisplayType.column,
+        ).joinWith(_TableRowPartDisplayType.columnSpacer);
+        rowDisplayParts.addAll(groupParts);
+        if (i < groups.length - 1) {
+          rowDisplayParts.add(_TableRowPartDisplayType.columnGroupSpacer);
+        }
+      }
+    } else {
+      final parts = List.generate(
+        widget.columns.length,
+        (_) => _TableRowPartDisplayType.column,
+      ).joinWith(_TableRowPartDisplayType.columnSpacer);
+      rowDisplayParts.addAll(parts);
+    }
+    return rowDisplayParts;
+  }
 
   @override
   void initState() {
     super.initState();
     contentKey = ValueKey(this);
     scrollController = widget.linkedScrollControllerGroup.addAndGet();
     _initSearchListeners();
+
+    rowDisplayParts = rowDisplayPartsHelper();
+    rowExtentDelegate = FixedExtentDelegate(
+      computeExtent: (index) {
+        final parts = rowDisplayParts;
```

Comment:
call rowDisplayPartsHelper here so that this is regenerated each time rowExtentDelegate.recompute is called